### PR TITLE
1957 - Laravel 8 Upgrade

### LINKED ISSUE #1957: Laravel 8 Upgrade

### PR DESCRIPTION
**Pull request by CalumTowers**
Resolves #1957

### CODE REVIEW

**Review by atoff — APPROVED** [review]

File: app/Models/Concerns/TracksChanges.php [hunk 1]
```diff
@@ -70,8 +70,6 @@ public function trackChanges()
         $attributes = $this->originalTracked;
 
         foreach ($this->trackableFromArray($attributes) as $key => $value) {
-            $key = $this->removeTableFromKey($key);
```
Comment (atoff) [inline app/Models/Concerns/TracksChanges.php:73]: Is this related to the update? I don't see this method in the file at all, not sure where it comes from

Comment (CalumTowers) [reply app/Models/Concerns/TracksChanges.php:73]: It has been deprecated for a while (ref: https://github.com/laravel/framework/pull/33865) and was removed in v8.0

Comment (CalumTowers) [reply app/Models/Concerns/TracksChanges.php:73]: Just double-tripe checked.
The method has, for a while, just simply returned the `$key` unchanged. It used to do the below;

```php
        if (! Str::contains($key, '.')) {
            return $key;
        }

        return last(explode('.', $key));
```

**Review by AxonC — APPROVED** [review]
Pleasing
